src/modules/xplorers-ai/xplorersAIHero.tsx

```diff
@@ -0,0 +1,26 @@
+import Image from "next/image";
+
+const XplorersAIHero = () => {
+  return (
+    <div className="grid grid-cols-2">
+      <Image
+        src="https://res.cloudinary.com/dumkkicev/image/upload/v1733812951/xplorers-ai/koo5k69yxpmaxqg21v5m.png"
```

Comment:
Not a big deal but would be great to save the source in some sort of constant ? may be a global constant.